src/blog/code-caching-for-devs.md

```diff
@@ -4,6 +4,7 @@ author: '[Leszek Swirski](https://twitter.com/leszekswirski), cache smasher'
 avatars:
   - leszek-swirski
 date: 2019-04-08 13:33:37
+updated: 2020-06-09 13:33:37
```

Comment:
Please update date to 16-06 (and probably remove time, that doesn't look like an actual time anyway?).

Reply:
Done.  I updated to "2020-06-16".  I removed the time.